pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,16 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	if inputBCNode, points := buildPointsToTag(g, tagNode); points {
```

Comment:
i still think this function needs to be updated to return all the BCs that point to the provided IST.  i guess it just returns the first one today?